people.json

```diff
@@ -460,5 +460,16 @@
     "twitter": "aminalipour_dev",
     "instagram": "",
     "telegram": ""
-  }
+  },
+  {
+    "name": "Milad Moafi",
+    "about": "Front-End Developer React.js, React-Native, Redux, MobX, PWA, SSR, Test",
```

Comment:
Please fill the name and about, in the Persian language.

Reply:
Done